src/DevBetterWeb.Infrastructure/Services/DiscordWebhookService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using DevBetterWeb.Infrastructure.DiscordWebhooks;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Infrastructure.Interfaces;
+
+namespace DevBetterWeb.Infrastructure.Services;
+
+public class DiscordWebhookService : IDiscordWebhookService
+{
+	private readonly HttpClient _httpClient = new();
+	private readonly IAppLogger<DiscordWebhookService> _logger;
+
+	public DiscordWebhookService(IAppLogger<DiscordWebhookService> logger)
+	{
+		_logger = logger;
+	}
+
+	public Task<HttpResponseMessage> SendAsync(ulong id, string token, string contentBody, string? username = null, string? avatarUrl = null, bool isTTS = false, IEnumerable<Embed>? embeds = null)
+	{
+		var webhookUrl = $"https://discordapp.com/api/webhooks/{id}/{token}";
+
+		return SendAsync(webhookUrl, contentBody, username, avatarUrl, isTTS, embeds);
+	}
```

Comment:
I believe this to be dead code; not being used or called anywhere. Judging by the pattern we have set, it is unlikely to ever be use. Don't forget YAGNI and KISS 😉.

Reply:
> I believe this to be dead code; not being used or called anywhere. Judging by the pattern we have set, it is unlikely to ever be used. Don't forget YAGNI and KISS 😉.

I already separated this into another project and take it from there. I think we can do the same and release it on NuGet later.
The same concept you are using here 
```csharp
public BaseWebhook(ulong id, string token, IAppLogger<BaseWebhook> logger) : this($"https://discordapp.com/api/webhooks/{id}/{token}", logger)
	{
	}
```

Reply:
I'm not sure I understand, I'm sorry. Could you help me understand how this method is being used in devBetter at the moment?